crates/store/re_types/definitions/rerun/blueprint/archetypes/line_grid_3d.fbs

```diff
@@ -0,0 +1,33 @@
+namespace rerun.blueprint.archetypes;
+
+/// Configuration for the 3D line grid.
+table LineGrid3D (
+    "attr.rerun.scope": "blueprint"
+) {
+    /// Whether the grid is visible.
+    ///
+    /// Defaults to true.
+    visible: rerun.blueprint.components.Visible ("attr.rerun.component_optional", nullable, order: 1000);
+
+    /// Space between grid lines spacing of one line to the next in scene units.
+    spacing: rerun.blueprint.components.GridSpacing ("attr.rerun.component_optional", nullable, order: 2000);
```

Comment:
The data model lgtm overall 👍🏻 The spacing is slightly ambiguous? Is that the small lines or big lines. For some reason, my (randomly picked) default mental model was that the spacing refers to the big lines, but looking at the screenshots it seems that it's for the small lines. Maybe just clarify that in the doc string.

And then we're good. (That is, until someone requests arbitrary sub-line count to cover for inch/feet/whatever.

Reply:
it's about the small lines. The big lines are right now a hardcoded visual gimmick to deal with zooming out, shows every 10th line
but yeah I can totally see where that confusion comes from!